src/EthernetServer.cpp

```diff
@@ -46,7 +46,7 @@ EthernetClient EthernetServer::available()
 	chip = W5100.getChip();
 	if (!chip) return EthernetClient(MAX_SOCK_NUM);
 #if MAX_SOCK_NUM > 4
-	if (chip == 51) maxindex = 4; // W5100 chip never supports more than 4 sockets
+	if (chip == 51 || chip == 50) maxindex = 4; // W5100 chip never supports more than 4 sockets
```

Comment:
Maybe the comment should be modified aswell to mention the W5100S aswell? Like "W5100(S)"